[WPE] Checkout newer version of COG

pgorszkowski-igalia opened the pull request:
#### c60dcc765e3b9226fa7e739e9221e06f35ac5b87
<pre>
[WPE] Checkout newer version of COG
<a href="https://bugs.webkit.org/show_bug.cgi?id=253962">https://bugs.webkit.org/show_bug.cgi?id=253962</a>

Reviewed by Adrian Perez de Castro.

Checkout the newer version of COG from the current
master tip which is: 48347f3a36e6dba75bcfd4f9443730861adec5b0

This fix includes also replacing meson option &apos;soup2&apos; with &apos;wpe_api&apos;
because of changes in COG.

* Tools/PlatformWPE.cmake:

Canonical link: <a href="https://commits.webkit.org/261704@main">https://commits.webkit.org/261704@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b0216045e43a68ac9d5fc51e2ae2646ab906214d

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/112581 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/21732 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/90/builds/1239 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/4357 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/121129 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/116647 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/23072 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/12894 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/5507 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/118348 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/17139 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/100349 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/105648 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/99093 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/88/builds/893 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/46164 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/14067 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/89/builds/929 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/95249 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/14750 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/10302 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/20091 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/52934 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/16587 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/4464 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

aperezdc reviewed (approved):
Thanks for fixing this, I keep forgetting that WebKit builds a copy of Cog, even if the build messages have been scrolling past my eyes for months 🙃 

(FTR, this was fallout caused by https://github.com/Igalia/cog/pull/550)

pgorszkowski-igalia committed:
[WPE] Checkout newer version of COG
https://bugs.webkit.org/show_bug.cgi?id=253962

Reviewed by Adrian Perez de Castro.

Checkout the newer version of COG from the current
master tip which is: 48347f3a36e6dba75bcfd4f9443730861adec5b0

This fix includes also replacing meson option 'soup2' with 'wpe_api'
because of changes in COG.

* Tools/PlatformWPE.cmake:

Canonical link: https://commits.webkit.org/261704@main
